ONSdigital/ssdc-rm-support-tool
# Validate print template when allowing on survey

# Motivation and Context
When we create print templates, we have no list of columns to validate against, so anything is allowed. We should instead validate at the point when we attempt to allow a print template on a particular survey.

# What has changed
Added validation at the point print templates are allowed on surveys.

# How to test?
Create a print template with crazy columns, then try to associate it to a survey... it should complain about the crazy columns.

# Links
Trello: https://trello.com/c/rUBC4RJt

# Screenshots (if appropriate):
<img width="592" alt="Screenshot 2021-08-12 at 12 01 02" src="https://user-images.githubusercontent.com/41681172/129186316-c86eddc4-0288-4d88-b6a2-c8cf3be6d25d.png">
